### DISCUSSION TITLE
Migration of Sunbird-mobile-app to Android SDK-33

### DISCUSSION THREAD

**Discussion by swayangjit** [Sunbird-Ed/Community#609] [General Discussion]
To meet Google Play’s [target API level requirements](https://support.google.com/googleplay/android-developer/answer/11926878).
Starting on August 31, 2023:

- New apps must target Android 13 (API level 33) or higher.

- App updates must target Android 13 or higher and adjust for behavioral changes in Android 13.
The documentation to migrate Sunbird-mobile-app to SDK-33 is available [here](https://project-sunbird.atlassian.net/wiki/spaces/SUN/pages/3354329090/Migration+of+Sunbird+app+from+to+android+SDK+level+31+to+SDK+level+33).

The patch to migrate to SDK-33 on top of release-5.1.0 is available in this [PR](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/pull/3479).
@aishwaryashikshalokam @Prateek-slokam @varsha-sl @MeghnaSL  @ashokreddy1208  @kameshbhr @mmanimegalai  @rajeevsathish @chitranshu-keshav @anisha147 @vinukumar-vs  @Krishnaj20 @kiranharidas187

**Comment by swayangjit** [comment]
This patch contains fixes for all Sunbird use cases. A few of the ML use cases are not working as expected so requesting the community to work on them.
@aishwaryashikshalokam @Prateek-slokam @varsha-sl @MeghnaSL @kiranharidas187

**Reply by kiranharidas187** [reply]
We are working on this. Will update here once the fix is made.

**Reply by kiranharidas187** [reply]
@swayangjit The fix for Manage Learn has been done in this [PR](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/pull/3494).
